cluster.py

```diff
@@ -1,21 +1,46 @@
 import logging
+import socket
 from pathlib import Path
-from typing import Dict
+from typing import Dict, Tuple
 
 from ccmlib import scylla_cluster as ccm
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
+def acquire_ip_prefix() -> Tuple[socket.socket, str]:
+    """gets unique ip prefix across whole machine,
+    so it's possible to run tests in parallel.
+
+    Returns tuple of lock (socket in that case) and ip prefix, where lock later needs to be released."""
+    logger.info("Getting machine-unique ip prefix to support parallel tests...")
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    for index in range(126):
```

Comment:
I would recommend starting the index from 1, and not from 0
i.e. so it would conflict with other tests the might use 127.0.0.1, like other driver tests, some specific dtest, or manual usage of ccm

Reply:
done